pymc/distributions/distribution.py

```diff
@@ -601,6 +656,22 @@ def update(self, node: Node):
         return updates
 
 
+@_moment.register(CustomSymbolicDistRV)
+def dist_moment(rv, *args):
+    x = args[0]
```

Comment:
@ricardoV94 I not very like this `args[0]` here and `args[1]` bellow, I mean magic numbers for index. Do you think we can avoid it somehow?